mne/beamformer/_lcmv.py

```diff
@@ -815,11 +821,17 @@ def lcmv_epochs(epochs, forward, noise_cov, data_cov, reg=0.05, label=None,
         if 'signed', the signed source space time series will be returned.
         'abs' is deprecated and will be removed in 0.16. Set max_ori_out to
         'signed' to remove this warning.
+
+    .. versionadded:: 0.15.0
+
     reduce_rank : bool
         If True, the rank of the leadfield will be reduced by 1 for each
         spatial location. Setting reduce_rank to True is typically necessary
         if you use a sphere model for MEG as in this case the actual
         rank is 2 not 3.
+
+    .. versionadded:: 0.15.0
+
```

Comment:
All of these are indented too much. And also you only need the newline before, not this extra one after

Reply:
Sorry, indented too little